.github/workflows/build-n-push_prjctr-mlops-hw2_image_to_docker-hub.yml

```diff
@@ -0,0 +1,31 @@
+name: Build and Push prjctr-mlops-hw2 image to Docker Hub
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  Build-n-Push_prjctr-mlops-hw2_image_to_Docker-Hub:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      - name: List files in the repository
+        run: |
+          ls ${{ github.workspace }}
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: ./containers/hw2/
+          file: ./containers/hw2/Dockerfile
+          push: true
+          tags: ${{ secrets.DOCKER_HUB_USERNAME }}/prjctr-mlops-hw2:latest
```

Comment:
please add caching here: https://github.com/truskovskiyk/ml-in-production-webinars/blob/main/.github/workflows/week-1-ci.yaml#L65-L66

Reply:
added
https://github.com/rl-illya13/prjctr-mlops/pull/3/commits/8b76069f9d0f193d964188d8877909208b02c02a